install_config/topics/gce_scopes_permissions.adoc

```diff
@@ -0,0 +1,82 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+[id='configuring-gcp-permissions_{context}']
+= Configuring authorization for Google Cloud Platform
+
+*Roles*
+
+Configuring GCP for {product-title} requires the following GCP role:
+
+[cols='1,3']
+|===
+
+|`roles/owner`
+|Needed for creating service accounts, cloud storage, instances, images,
+templates, Cloud DNS entries, and to deploy load balancers and health checks.
+|===
+
+`delete` permissions might also be required if the user is expected to be
+redeploying the environment during testing phases.
+
+You can also create a service account to avoid using personal users when
+deploying GCP objects.
+
+See link:https://cloud.google.com/iam/docs/understanding-roles[the Understanding roles section of the GCP documentation] for more information, including steps for how to configure roles.
+
+*Scopes and service accounts*
+
+GCP uses scopes to determine if an authenticated identity is authorized to
+perform operations within a resource. For example, if application A with a
```

Comment:
Wait. Can the user read/write or the application read/write? It sounds like the application is doing the read/write actions in the second sentence, but it sounds like an identity (user?) is performing the operations in sentence 1.

Reply:
Users or applications, it doesn't matter (they are 'authenticated identities')